openlibrary/macros/BookByline.html

```diff
@@ -1,21 +1,26 @@
-$def with(author_names_and_urls, limit=None, overflow_url=None, attrs='')
+$def with(author_names_url_birth_date_death_date, limit=None, overflow_url=None, attrs='')
 
-$def render_author_link(name, url):
+$def render_author_link(name, url, birth_date, death_date):
   $ rendered_name = cond(name, truncate(name, 40), _("Unknown author"))
+  $ birth_date_formatted = ''
+  $ death_date_formatted = ''
+  $if birth_date and death_date:
+    $ birth_date_formatted = birth_date[-4:]
```

Comment:
We should be using a year regex here, since our dates are sometimes in weird formats